radix-engine-common/src/math/decimal.rs

```diff
@@ -98,20 +98,20 @@ impl Decimal {
 
     /// Returns the largest integer that is equal to or less than this number.
     pub fn floor(&self) -> Self {
-        self.round(0, RoundingMode::TowardsNegativeInfinity)
+        self.round(0, RoundingMode::ToNegativeInfinity)
     }
 
     /// Returns the smallest integer that is equal to or greater than this number.
     pub fn ceiling(&self) -> Self {
-        self.round(0, RoundingMode::TowardsPositiveInfinity)
+        self.round(0, RoundingMode::ToPositiveInfinity)
     }
 
     pub fn round(&self, decimal_places: u32, mode: RoundingMode) -> Self {
```

Comment:
I wonder if we want `round(&self, decimal_places: impl Into<i32>, mode: RoundingMode)`, and allow negative decimal places?

Reply:
Separately, this method asserts on decimal_places - could we add this to the Rust doc as a clear precondition for calling this?

Reply:
I will change to `i32` and leaving support of negative decimal places to future.